app/helpers/rules_of_origin_helper.rb

```diff
@@ -0,0 +1,21 @@
+module RulesOfOriginHelper
+  def rules_of_origin_service_name
+    TradeTariffFrontend::ServiceChooser.uk? ? 'UK' : 'EU'
+  end
+
```

Comment:
we use this logic in here
`app/views/rules_of_origin/_tab.html.erb#15`  too,
maybe we can reuse this method there

Reply:
:) Just found that - I've changed it in #257